demo/src/app/components/modal/modal.component.ts

```diff
@@ -16,6 +16,9 @@ import {DEMO_SNIPPETS} from './demos';
       <ngbd-example-box demoTitle="Components as content" [snippets]="snippets" component="modal" demo="component">
           <ngbd-modal-component></ngbd-modal-component>
       </ngbd-example-box>
+      <ngbd-example-box demoTitle="Dismiss all modal windows" [snippets]="snippets" component="modal" demo="dismissall">
+          <ngbd-modal-dismissall></ngbd-modal-dismissall>
+      </ngbd-example-box>
```

Comment:
I'm not sure if we need a dedicated demo for this new feature - I would rather drop it as IMO it doesn't add enough value.